### DISCUSSION TITLE
Quick 'n dirty checklist for designing a meeting

### DISCUSSION THREAD

**Discussion by Emmy-Emberverse** [GECO-Project-App/facilitation-app#75] [1 - Ideas, Suggestions, & Requests]
https://www.chriscorrigan.com/parkinglot/simple-meeting-design/

I imagine this could either be a piece of information somewhere, or a meeting design template

**Comment by eliasfaltin** [comment]
This is a good idea.
I was playing with the idea of letting people create "custom agendas" where they can collect and order exercises in "agendas" to be re-used (or even shared ahead of time with the team).

This flow could include text input fields and prompts/explanations for:

- Purpose
What is the big purpose that we are trying to fulfill?

- Harvest (Outcome)
**One could argue that this point is a bit too "kumbaya" for most.**
What do you want to harvest? **(What do we want the outcome of this meeting to be?)**
In our hands ( tangible)? **(What tangible outcomes do we want?)**
In our hearts ( intangible)? **(What intangible outcomes do we want? How do we want to feel when leaving the meeting?)**

- ~~Wise~~ Action (How do we make the desired outcome happen during the meeting?)
**This is where users would select exercises from the catalog to build their "action" sequence for a given meeting.**
How will we make action happen?
Who will help us tune in to the reality of the situation?
How will you keep people together?

- Invitation
**This is where we could implement a "share" function that generates a view-only link for the meeting agenda that the facilitator can send to the team. Plus, the agenda could be "scheduled" and automagically appear in the app for team members**
What is the inspiring question that will bring people together?
How will we invite people so they know they are needed?
